examples/ui-demo/tailwind.config.ts

```diff
@@ -2,7 +2,7 @@ import { withAccountKitUi } from "@account-kit/react/tailwind";
 import type { Config } from "tailwindcss";
 
 const config = {
-  darkMode: ["class"],
+  darkMode: "selector",
```

Comment:
oh nice! does this ultimately allow you to do dark mode on sub trees or does this just enable using `dark` as a class?

Reply:
no, this just replaces the "class" version in latest tailwind:
https://tailwindcss.com/docs/dark-mode#toggling-dark-mode-manually